src/compiler/cell_manager.rs

```diff
@@ -330,6 +332,20 @@ impl CellManager for MaxWidthCellManager {
             placement.steps.insert(step.uuid(), step_placement);
         }
 
+        if self.same_height {
+            let height = placement
+                .steps
+                .iter()
+                .map(|(_, step)| step.height)
+                .max()
+                .unwrap_or(0);
```

Comment:
How about doing `expect` here to panic with an error message instead of `unwrap_or`?

Reply:
Just saying height = 0 doesn't make much sense in this context.

Reply:
This will return None, ony if there is no steps, in a circuit without steps, 0 step height makes sense. Perhaps we can revisit this if we have circuits with only fixed signals.

Reply:
kk sounds good to me!